tests/integration/docusaurus/connecting_to_your_data/cloud/azure/pandas/partitioned_on_datetime.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
I'm not sure this should be in the cloud directory, but putting it here to colocate it with its related tests.